### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by NolanReda**
mobile layout 1
moble layout 2
desktop blank
desktop filled out

### CODE REVIEW

**Comment by NolanReda** [comment]
local storage for feature 1

**Comment by NolanReda** [comment]
mobile layout 1
mobile layout 2

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
+body {
+  max-width: 1000px;
+  margin: auto;
```
Comment (codingcodymiller) [inline css/styles.css:3]: You can remove these, the `container` class will handle this for you.

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  background-color: #f0f0f0;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+
+header {
+  height: 3.5rem;
```
Comment (codingcodymiller) [inline css/styles.css:9]: You shouldn't set height on elements unless it is absolutely necessary.

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  border-radius: 4px;
+}
+
+.pad-top-5 {
+  padding-top: 5rem;
+}
+
+/* .marg-right-15 {
+  margin-right: 15rem;
+} */
```
Comment (codingcodymiller) [inline css/styles.css:65]: Please delete commented code prior to submitting for review.

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+$urlInput.addEventListener('input', changeSrc);
+
+function handleSubmit(event) {
+  var newEntry = {};
+  newEntry.image = $urlInput.value;
+  newEntry.title = $title.value;
+  newEntry.notes = $notes.value;
+  newEntry.nextEntryId = data.nextEntryId;
+  data.entries.push(newEntry);
+  data.nextEntryId++;
```
Comment (codingcodymiller) [inline js/main.js:22]: You are not preventing the default behavior of the form refreshing the page, and you are not resetting the form.

File: js/main.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+//   text.setAttribute('class', 'column-half');
+//   var title = document.createElement('h2');
+//   var h2 = createTextNode(data.entries.title);
+//   title.appendChild(h2);
+//   var notes = document.createElement('p');
+//   var p = document.createTextNode(data.entries.notes);
+//   notes.appendChild(p);
+// }
+
+// var div = document.querySelector('main');
```
Comment (codingcodymiller) [inline js/main.js:83]: Why do you have code for the next feature? Please delete this, you should not be working ahead.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**